testenrichers/cdi/src/test/java/org/jboss/arquillian/testenricher/cdi/CDIInjectionEnricherTestCase.java

```diff
@@ -137,77 +113,6 @@ public void shouldInjectMethodArgumentsInstance() throws Exception {
         testMethod.invoke(testClass, resolvedBeans);
     }
 
-    private Deployment createDeployment(final Class<?>... classes) {
```

Comment:
Thanks for that simplification!

Reply:
Or is that something to be reverted @manovotn ?

Reply:
It should be reverted if this is going to run against cdi 1.0 api